installer/bundle/bundle_skel.sh

```diff
@@ -871,10 +910,7 @@ case "$installMode" in
 
             pkg_add_list $OMS_PKG omsagent
 
-            python_ctypes_installed
-            if [ $? -eq 0 ]; then
-                pkg_add_list $DSC_PKG omsconfig
-            fi
+            if shouldInstall_omsconfig; then pkg_add_list $DSC_PKG omsconfig; fi
```

Comment:
We should standardize the single-line if statement vs. mutli-line if block throughout the code; as it is, this looks fine.